tidb-cloud/data-service-postman-integration.md

```diff
@@ -0,0 +1,93 @@
+---
+title: Run Data App in Postman
+summary: Learn how to run your Data App in Postman.
+---
+
+# Run Data App in Postman
+
+[Postman](https://www.postman.com/) is an API platform for building and using APIs, which simplifies each step of the API lifecycle and streamlines collaboration so you can create better APIs—faster.
```

Comment:
Postman is an API platform that simplifies the API lifecycle and enhances collaboration for faster and better API development